src/crypto/kem/kem.go

```diff
@@ -0,0 +1,203 @@
+package kem
```

Comment:
It is not a one to one wrapper. What this package does is that it exposes an interface that is used by tls based on the kem functions. I can move this code to its own file on the tls folder; but it seemed a little weird to have it there.

Reply:
What about moving it `crypto/kem` to `crypto/tls/internal/kem`?

Reply:
As codeowner for `crypto/tls`, you would be codeowner of this internal package. Otherwise, we would need to manually assign you as codeowner of `crypto/kem`.

Reply:
> Otherwise, we would need to manually assign you as codeowner of crypto/kem.

And what is the problem with this?

Reply:
Leaving this discussion for next next week.